src/application.rs

```diff
@@ -100,7 +100,7 @@ pub trait Application: Default + Sized {
         // Load configuration
         let config = command
             .config_path()
-            .map(|_| self.load_config(command))
+            .map(|path| command.process_config(self.load_config(&path)?))
```

Comment:
What happens if `load_config` fails? Does the error bubble up to exit the program, or does it just not load the config (cf. #80)?

Reply:
It bubbles up here, and the application exits:

https://github.com/iqlusioninc/abscissa/blob/develop/src/application.rs#L58

If you'd like for a nonexistent config file to be ignored, the command's `config_path()` needs to check for the file's existence in advance, and if it's not there, return `None`

Reply:
Hmm, I'm confused because I thought from https://github.com/iqlusioninc/abscissa/issues/80#issuecomment-509429594 that the intent was that `config_path` doesn't need to check. (Either way is fine, as long as it's clear one way or the other).

Reply:
It doesn't need to check if you're fine with the application exiting with an error if the config doesn't exist.

If you want an absent config to be ignored, `config_path()` needs to return `None`